pyredlight/fastapi.py

```diff
@@ -7,14 +8,28 @@ async def default_get_key(request: Request):
     return request.client.host
 
 
-def make_depends(limiter: Limiter, get_key=default_get_key):
+def make_depends(
+    limiter: Limiter,
+    get_key=default_get_key,
+    retry_after_header: bool = False,
```

Comment:
```suggestion
    retry_after_header: bool = True,
```
We don't really have to worry about backwards compatibility, and if we did, we could just use version numbers for that.